sw/host/sphincsplus/BUILD

```diff
@@ -0,0 +1,24 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@rules_rust//bindgen:defs.bzl", "rust_bindgen_library")
+load("@rules_rust//rust:defs.bzl", "rust_library", "rust_test")
+
+rust_bindgen_library(
+    name = "sphincsplus_bindgen",
+    cc_lib = "@sphincsplus_fips205_ipd//:sphincs_random_shake_128s_simple",
+    clang_flags = ["-DPARAMS=sphincs-shake-128s"],
+    header = "@sphincsplus_fips205_ipd//:api.h",
```

Comment:
The library generated by this rule is pretty permissive about what gets bindings.  You should use some flags to be more restrictive (ie: we don't care about most of the junk that gets incidentally included like `uint128_t` or `foff_t`, etc).

You could try something like this:
```
    bindgen_flags = [
        "--allowlist-function=crypto_.*",
        "--allowlist-var=CRYPTO_.*",
        "--allowlist-var=SPX_.*",
    ],
```
(see also the [bindgen manual](https://rust-lang.github.io/rust-bindgen/allowlisting.html))

Reply:
From a cursory look at what gets generated, this looks fine.